examples/latte/README.md

```diff
@@ -213,11 +230,12 @@ The first number `0` indicates the start index of the training devices, and the
 
 The training speed of the experiments with `256x256` image size is summarized in the following table:
 
-| Cards | Recompute | Dataset Sink mode | Embedding Cache|Train. imgs/s |
-| ---   | ---       | ---               | ---          |   ---          |
-| 1     | OFF       | ON                | OFF          | 62.3           |
-| 1     | ON        | ON                | ON           | 93.6           |
-| 4     | ON        | ON                | ON           | 368.3          |
+| model name |   context    | cards | image size | graph compile |  bs   | recompute | sink | cache | step time | train. imgs/s |
+| :--------: | :----------: | :---: | :--------: | :-----------: | :---: | :-------: | :--: | :---: | :-------: | :-----------: |
+|   latte    | 910*-MS2.3.1 |   1   |  256x256   |   6~8 mins    | 5x16  |    OFF    |  ON  |  OFF  |   1.03s   |     77.67     |
+|   latte    | 910*-MS2.3.1 |   1   |  256x256   |   6~8 mins    | 1x128 |    ON     |  ON  |  ON   |   1.21s   |    105.78     |
+|   latte    | 910*-MS2.3.1 |   4   |  256x256   |   6~8 mins    | 1x128 |    ON     |  ON  |  ON   |   1.32s   |    387.87     |
+|   latte    | 910*-MS2.3.1 |   8   |  256x256   |   6~8 mins    | 1x128 |    ON     |  ON  |  ON   |   1.31s   |    781.67     |
```

Comment:
Change `bs` to two columns: bs and num_frames. Maybe also explain `sink` at the bottom of the table? I'm afraid the common users will not know about data sink.

Reply:
resolved